### ISSUE TITLE
Add Open Graph metadata to ensure maximum fidelity for LinkedIn shares

### ISSUE THREAD

**Issue by pcgeek86** [balena-io/docs#1080]
## Expected Result

Preview image shows up for balena documentation pages, when they're shared on LinkedIn. A balena logo would suffice, but a more context-specific image preview would also be welcome.

## Actual Result

No preview image appears for documentation pages shared on LinkedIn.

![image](https://user-images.githubusercontent.com/466713/56157849-6877ba00-5f75-11e9-89a3-5eed3fbfd61f.png)

## Action Items

* Implement Open Graph HTML meta tags, based on [LinkedIn requirements](https://www.linkedin.com/help/linkedin/answer/46687/making-your-website-shareable-on-linkedin?lang=en)
  * Max file size: 5 MB
  * Minimum image dimensions: 1200 (w) x 627 (h) pixels
  * Recommended ratio: 1.91:1

### Example Open Graph Meta Tags

```
<meta property='og:title' content='Title of the article"/>
<meta property='og:image' content='//media.example.com/ 1234567.jpg"/>
<meta property='og:description' content='Description that will show in the preview"/>
<meta property='og:url' content='//www.example.com/URL of the article" />
```

**Comment by pcgeek86** [comment]
Looks like this tag exists, but the image is missing from the `resin-site` repo. 

```
<meta property="og:image" content="/img/docs-preview.png">
```

https://www.balena.io/img/docs-preview.png - 404

**Comment by dimitrisnl** [comment]
The fact that it's targeting the main website is a mistake. The docs used to be at `docs.resin.io`, so it's incorrectly not prefixed.

**Comment by pcgeek86** [comment]
Resolved in #1081
